stockpile/roles/openshift-cluster-topology/files/openshift_config_scraper.py

```diff
@@ -0,0 +1,43 @@
+import yaml
```

Comment:
we can have the env decide which version of python to run, adding (#!/usr/bin/env python) should do and perhaps make the code python3 friendly